apps/site/package.json

```diff
@@ -18,7 +19,12 @@
     "sync-orama": "node ./scripts/orama-search/sync-orama-cloud.mjs",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 node --experimental-test-coverage --test-coverage-exclude=**/*.test.* --experimental-test-module-mocks --enable-source-maps --import=global-jsdom/register --import=tsx --import=tests/setup.jsx --test **/*.test.*",
     "test:unit:watch": "cross-env NODE_OPTIONS=\"--watch\" npm run test:unit",
-    "test": "turbo test:unit"
+    "test": "turbo test:unit",
+    "build-blog-data": "node ./scripts/blog-data/generate.mjs",
+    "build-blog-data:watch": "node --watch --watch-path=pages/en ./scripts/blog-data/generate.mjs",
+    "cloudflare:build:worker": "npx @opennextjs/cloudflare build",
```

Comment:
Is there a bin from node_modules we can call, or does npx have to be used?

Reply:
`npx` will use the existing node_modules version if it exists? I think calling to a node_modules bin directly is rather fragile compared to using `npx`?

Reply:
I don't think it's consistent. We don't use `npx turbo` in the other scripts.

Reply:
Ah true, we could probably call `opennextjs-cloudflare` directly

Reply:
yes this is simply calling the node_modules one, sorry I don't remember why I used `npx` here and I must have forgotten to check wether this is consistent with the codebase, I'll remove the `npx bit` 🙇

Reply:
`npx` removed 🙂